bazarr.py

```diff
@@ -31,20 +32,20 @@ def check_python_version():
 dir_name = os.path.dirname(__file__)
 
 
-def end_child_process(ep):
+def terminate_child_process(ep):
     try:
-        ep.kill()
+        ep.terminate()
```

Comment:
This may be actually a personal preference. But from what I gathered the `TERM` signal allows for resource cleanup while the `KILL` is pretty "definite". At least on Windows.